src/plugins/intel_npu/src/utils/include/intel_npu/utils/zero/zero_graph_ext_wrappers.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2018-2024 Intel Corporation
```

Comment:
Why move it to utils? It is needed only for the graph that is part of the compiled model, right?

Reply:
the utils will be wrapper for zero relate things, and we can add tests easily for ext with this. But put it inside compiler adapter also work for me